analysis/mc_validation/mc_validation_gen_processor.py

```diff
@@ -171,16 +177,16 @@ def process(self, events):
             event_weight_cut = event_weight[isnotnone_mask]
             eft_coeffs_cut = eft_coeffs
             if eft_coeffs is not None: eft_coeffs_cut = eft_coeffs[isnotnone_mask]
-            eft_w2_coeffs_cut = eft_w2_coeffs
-            if eft_w2_coeffs is not None: eft_w2_coeffs_cut = eft_w2_coeffs[isnotnone_mask]
+            #eft_w2_coeffs_cut = eft_w2_coeffs
+            #if eft_w2_coeffs is not None: eft_w2_coeffs_cut = eft_w2_coeffs[isnotnone_mask]
```

Comment:
Wondering why you commented out these 2 lines (180 and 181). Does this have something to do with the new hist changes?

Reply:
Yes, we no longer store the errors for EFT samples. The workaround I did on the master branch (having a second histogram for the of the square of the weights) should eventually go here.

Reply:
Ah okay. Sorry for the noise.